File: PooCC01CNetz/Semana04/Java/Ejercicios01/Ciudadano.java
public class Ciudadano {
    private String nombre;
    private Domicilio direccion;

    public Ciudadano(String nombre ,Domicilio direccion) {
        this.nombre = nombre;
        this.direccion = direccion;
    }

    public Ciudadano() {}

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setDomicilio(Domicilio direccion) {
        this.direccion = direccion;
    }

    public Domicilio getDomicilio() {
        return direccion;
    }
}

File: PooCC01CNetz/Semana04/Python/Ejercicios01/Ciudadano.py
from Domicilio import *
class Ciudadano:
    def __init__(self, nombre = None, direccion = None):
        # Constructor vacio
            # Atributos privados
        if nombre is None and direccion is None:
            self.__nombre = nombre
            self.__direccion = direccion
            pass
        else:
             # Constructor con parámetros
            self.__nombre = nombre
            self.__direccion = direccion


    def setNombre(self, nombre):
        self.__nombre = nombre

    def getNombre(self):
        return self.__nombre
    
    def setDomicilio(self, direccion):
        self.__direccion = direccion

    def getDomicilio(self):
        return self.__direccion